front-end/package.json

```diff
@@ -18,6 +18,7 @@
     "dateformat": "^4.5.1",
     "node-sass": "^4.14.1",
     "react": "^17.0.1",
+    "react-bootstrap": "^1.5.2",
```

Comment:
Is there a reason you installed react-bootstrap? Doesn't look like it's being used anywhere. We're using reactstrap and having react-bootstrap might cause problems

Reply:
yes I was trying to do something with the UI but I ended up not using it. I'll remove it.

Reply:
Actually we need it for App.scss

Reply:
App.scss isn't used so it seems like you can delete it.